web/portal/user/src/translations/it.json

```
@@ -3,6 +3,7 @@
   "Active call_one": "Chiamata attiva (uno)",
   "Active call_other": "Chiamate attive (altro)",
   "Answered": "Risposto",
+  "Basic Information": "Basic Información",
```

```suggestion
  "Basic Information": "Basic Information",
```
or `Informazioni di base`